api/types/types.proto

```diff
@@ -377,8 +410,32 @@ message CertAuthoritySpecV2 {
     }
     SigningAlgType SigningAlg = 9 [ (gogoproto.jsontag) = "signing_alg,omitempty" ];
     // JWTKeyPair is a list of JWT key pairs.
-    repeated JWTKeyPair JWTKeyPairs = 10
-        [ (gogoproto.nullable) = false, (gogoproto.jsontag) = "jwt_key_pairs,omitempty" ];
+    //
+    // DEPRECATED: use ActiveKeys and AdditionalTrustedKeys instead.
+    repeated JWTKeyPair JWTKeyPairs = 10 [
+        (gogoproto.nullable) = false,
+        (gogoproto.jsontag) = "jwt_key_pairs,omitempty",
+        deprecated = true
+    ];
+
+    // ActiveKeys are the CA key sets used to sign any new certificates.
+    CAKeySet ActiveKeys = 11
+        [ (gogoproto.nullable) = false, (gogoproto.jsontag) = "active_keys,omitempty" ];
+    // AdditionalTrustedKeys are additional CA key sets that can be used to
+    // verify certificates. Certificates should be verified with
+    // AdditionalTrustedKeys and ActiveKeys combined.
+    CAKeySet AdditionalTrustedKeys = 12
```

Comment:
`Additional` is here to avoid confusing the caller trying to validate certificates.
To properly validate certs, the caller must use both `ActiveKeys` and `AdditionalTrustedKeys`. If this field was called `TrustedKeys`, the caller might mistakenly assume that these are the only CA keys used for validation.

Updated the RFD.